test/next/src/app/vercel/blob/handle-blob-upload.ts

```diff
@@ -36,7 +36,7 @@ export async function handleBlobUploadHandler(
         }
 
         return {
-          maxFileSize: 10_000_000,
+          maximumSizeInBytes: 10_000_000,
```

Comment:
don't we need to update `https://github.com/vercel/edge-functions` accordingly?

Reply:
edge-functions is already accepting maximumSizeInBytes, I think at some point we renamed this and it did not rename it everywhere

Reply:
ah nice! I only checked your last PR, but indeed I see the name is `maximumSizeInBytes`👍